pyproject.toml

```diff
@@ -69,7 +69,7 @@ tests = [
     "pyansys-tools-report==0.6.0",
     "vtk==9.2.6",
     "pytest-rerunfailures==12.0",
-    "pytest-pyvista==0.1.8",
+    "pytest-pyvista@git+https://github.com/pyvista/pytest-pyvista#main",
```

Comment:
You won't be able to perform a PyPI release until this is solved... you know that, right?

Reply:
Oh... I didn't know that... Thanks God they just did one :)